app/descriptors/budget/budget2dig/subtitle-template.html

```diff
@@ -1,5 +1,5 @@
 סעיף תקציבי בתקציב המדינה - תחת תת-הנושא
-<a href="/i/budget/C{{(func_cls_json[0] | load_json)[0]}}{{(func_cls_json[0] | load_json)[2]}}/{{year}}">
+<a href="{{ 'budget' | item_link('C'+(func_cls_json[0] | load_json)[0]+(func_cls_json[0] | load_json)[2] + '/' + year, theme_id) }}">
```

Comment:
do we need to add theme_id every time? 😢

Reply:
I had some problems due to the way these templates are rendered, I could try again

Reply:
since it's a global value in the app, it should be possible to avoid it I think